src/docs/product/cli/dif.mdx

```diff
@@ -273,12 +273,12 @@ sentry-cli upload-proguard \
 ```
 
 Since the Android SDK needs to know the UUID of the mapping file, you need
-to embed it in a `sentry-debug-meta.properties` file. If you supply
-`--write-properties` that is done automatically:
+to associate it with the upload. If you supply
+`--uuid` sets that value:
 
 ```bash
 sentry-cli upload-proguard \
-    --write-properties app/build/generated/assets/sentry/{BuildVariant}/sentry-debug-meta.properties \
+    --uuid A_VALID_UUID \
     app/build/outputs/mapping/{BuildVariant}/mapping.txt
 ```
 
```

Comment:
Between the bash script and the next line, we'd need to tell users how to associate the uploaded `id` with the Android SDK, See https://github.com/getsentry/sentry-docs/issues/4431 for a code snippet

Reply:
Will add that info, as well as the requirement for UUID to be unique for each mapping (https://github.com/getsentry/sentry-cli/issues/914#issuecomment-1050585835)

Reply:
@btseytlinTCP can you make a new PR addressing the pending point? this PR was early merged.

Reply:
Oh my bad, fixed it https://github.com/getsentry/sentry-docs/pull/4771

Reply:
Thank you all!